tractor/_actor.py

```diff
@@ -1123,7 +1135,11 @@ async def cancel(self) -> bool:
 
             # stop channel server
             self.cancel_server()
-            await self._server_down.wait()
+            if self._server_down is not None:
+                await self._server_down.wait()
```

Comment:
Handles the case where the TCP server task never comes online before a cancellation request is serviced from the parent.